QBot/telegram_bot/views/sign_up.py

```diff
@@ -0,0 +1,81 @@
+from telegram import Update, ReplyKeyboardMarkup
+from telegram.ext import CallbackContext, ConversationHandler, Filters, MessageHandler
+from .start import reply_keyboard
+from .error import wrong_message, cancel
+from ..models import TelegramUser
+
+BASE = 0
+
+
+def signing_up(update: Update, context: CallbackContext):
+
+    reply_keyboard = [["with Telegram account", "custom sign up"], ["Cancel"]]
+    chat_id = update.message.from_user.id
+    context.bot.send_message(
+        chat_id,
+        text="options:",
+        reply_markup=ReplyKeyboardMarkup(reply_keyboard, one_time_keyboard=True),
+        parse_mode="Markdown",
+    )
+    return BASE
+
+
+def default_create_user(update: Update, context: CallbackContext):
+    try:
+        chat_id = update.message.from_user.id
+        username = update.message.from_user.username
+        first_name = update.message.from_user.first_name
+        last_name = update.message.from_user.last_name or None
+        is_bot = update.message.from_user.is_bot
+        language_code = update.message.from_user.language_code
+
+        TelegramUser.objects.create(
```

Comment:
SignUp will handle by the User model!

we will save all the Telegram account which are using the bot :)

Logged in User = a User object with a foreign-key to a TelegramUser!